mods/key_value.js

```diff
@@ -255,7 +266,8 @@ KVBucket.prototype.putRevision = function(restbase, req) {
                 key: rp.key,
                 tid: tid,
                 value: req.body,
-                headers: req.headers
+                headers: req.headers,
+                'content-type': req.headers && req.headers['content-type']
```

Comment:
Because since we cannot remove the field in the DB, let's populate it - maybe it'll be made useful in a next iteration (categorisation of values, e.g.)